aiohttp/client_ws.py

```diff
@@ -141,6 +140,15 @@ def _send_heartbeat(self) -> None:
         self._cancel_pong_response_cb()
         self._pong_response_cb = loop.call_at(when, self._pong_not_received)
 
+        ping_task = create_eager_task(self._writer.ping(), loop)
```

Comment:
ping task now happens at the end to better align with previous implementation since it previously would run one event loop iteration later on older python.